server/graphql/invoice/src/mutations/inbound_shipment/update.rs

```diff
@@ -32,6 +42,8 @@ pub struct UpdateInput {
     pub tax: Option<TaxInput>,
     pub currency_id: Option<String>,
     pub currency_rate: Option<f64>,
+    pub default_donor_id: Option<NullableUpdateInput<String>>,
+    pub update_donor_method: Option<UpdateDonorMethodInput>,
```

Comment:
This method is to allow for conditional logic of updating the child donor lines of the inbound shipment when we change the default donor